include/cinatra/coro_http_client.hpp

```diff
@@ -252,6 +252,10 @@ class coro_http_client {
           return false;
       }
 
+      if (base_path.empty() && cert_file.empty()) {
+        ssl_ctx_->set_default_verify_paths();
```

Comment:
Why we need to call this set_default_verify_paths()? is there any problem?

Reply:
details in https://gitlab.com/eidheim/Simple-WebSocket-Server/-/blame/master/client_wss.hpp#L51.
when user not have cert, let they use openssl default cert.set_default_verify_paths will tell context search cert file in openssl default path.